src/app/services/api-file.service.ts

```diff
@@ -200,6 +196,41 @@ export class ApiFileService {
     );
   }
 
+  getAllAnalyses(query: any, size: number) {
+    const url = 'http://wp-np3-e2:9200/faang_build_1_analysis/_search/' + '?size=' + size;
+//    const url = this.hostSetting.host + 'analysis/' + '_search/' + '?size=' + size;
+    const params = new HttpParams().set('_source', query['_source'].toString()).set('sort', query['sort']);
+    return this.http.get(url, {params: params}).pipe(
+      map((data: any) => {
+        return data.hits.hits.map( entry => ({
+          accession: entry['_source']['accession'],
+          datasetAccession: entry['_source']['datasetAccession'],
+          title: entry['_source']['title'],
+          species: entry['_source']['organism']['text'],
+//          assayType: this.convertArrayToString(entry['_source']['assayType']),
```

Comment:
Don't forget to uncomment it when warehouse will be ready.

Reply:
updated in bde3feb